app/views/events/archive.html.haml

```diff
@@ -1,3 +1,7 @@
 - set_title t("events.archive.title")
+= render 'search_form'
```

Comment:
I've rewritten the search controller to include archived events. See events.rb search